### PR TITLE
Update README.md: added my project

### PR DESCRIPTION
**Pull request by sbond75**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,25 @@ See other files for getting started documentation. Current projects:
 ## Ghidra
 * Reword last sentence of README.md so that it's understandable
 
+## Urho3D (instead of Godot)
```
Comment (mkenigs) [inline README.md]: Can you get rid of everything in parenthesis

**Review by mkenigs — CHANGES_REQUESTED** [review]
I'll merge it after you make that one change!

**Comment by mkenigs** [comment]
Thanks for being so thorough, looks great!

**Comment by sbond75** [comment]
Did you mean to remove all parenthesis in the whole thing or just in the title?

**Comment by mkenigs** [comment]
Just the instead of Godot, sorry that wasn't clear

**Pull request merged**